LayoutTests/imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/form-controls/resets.html

```diff
@@ -57,9 +57,7 @@
 tfoot, tfoot[hidden] { display: table-footer-group; }
 tr, tr[hidden] { display: table-row; }
 td, th { display: table-cell; }
-table {
-  text-indent: initial;
-}
+table { text-indent: initial; }
```

Comment:
nit: please undo this change completely since it's imported (and unlist this file from the commit message as well)

Reply:
huh. Why did I do that. Ah… yes I removed the border-box I had added in there. Sorry about that.